plugins/org.jkiss.dbeaver.ext.mysql/src/org/jkiss/dbeaver/ext/mysql/model/plan/MySQLPlanAnalyser.java

```diff
@@ -120,33 +118,19 @@ public void addNodeProperties(DBCPlanNode node, JsonObject nodeJson) {
                     attributes.add("extra", new JsonPrimitive(CommonUtils.notEmpty(plainNode.getExtra())));
                 } else if (node instanceof MySQLPlanNodeJSON) {
                     MySQLPlanNodeJSON jsNode = (MySQLPlanNodeJSON) node;
-                    for(Map.Entry<String, String>  e : jsNode.getNodeProps().entrySet()) {
-                        attributes.add(e.getKey(), new JsonPrimitive(CommonUtils.notEmpty(e.getValue())));
+                    for(Map.Entry<String, Object>  e : jsNode.getNodeProps().entrySet()) {
+                        Object value = e.getValue();
+                        if (value instanceof Double) {
+                            attributes.add(e.getKey(), new JsonPrimitive((Double) value));

```

Comment:
It would be useful to know why do we need this special workaround for double here, but don't in other places. Could you leave some comment?

Reply:
For me, it was clear but ok.

Reply:
Do we need to test mysql execution plan also then?

Reply:
Yes. It is on the list.